src/Passes/Source/Commandline/Settings.hpp

```diff
@@ -0,0 +1,68 @@
+#pragma once
```

Comment:
I like the use of pragma once when possible.

Reply:
This is actually enforced by CI - you are not allowed to make headers that does not have this atm.